schema.graphql

```diff
@@ -2045,6 +2068,7 @@ type Asset @entity {
   id: ID! # ticker
   ticker: String! @index(unique: true)
   name: String @index(unique: false)
+  isNftCollection: Boolean!
```

Comment:
IMO we should add index here `@index(unique: false)` since this would be used a lot in querying the data

Reply:
Also, do you think we should have this type of boolean parameter in `AssetTransaction` as well?

Reply:
I don't think a boolean index helps that much, it doesn't really organize data much since so many things are tied.

I found this SO that has some discussion around it: https://stackoverflow.com/questions/2113181/does-it-make-sense-to-use-an-index-that-will-have-a-low-cardinality

Reply:
I was thinking people would look at the asset to check for nft or not.

I guess its a bit easier to query having it inline, but also seems redundant. Most queries for `AssetTransaction` probably still need to grab the name and ticker from `Asset` anyway.